cmd/image-builder/main.go

```diff
@@ -520,12 +516,21 @@ operating systems like Fedora, CentOS and RHEL with easy customizations support.
 
 	rootCmd.AddCommand(describeImgCmd)
 
+	verbose, err := rootCmd.PersistentFlags().GetBool("verbose")
+	if err != nil {
+		return err
+	}
+	if verbose {
+		olog.SetDefault(log.New(os.Stderr, "", 0))
+		// XXX: add once images has olog support
+		//images_log.SetDefault(log.New(os.Stderr, "", 0))
```

Comment:
We can enable this now (as the  images PR is now merged), right? But I guess this can also be a followup

Reply:
Oh great eyes, yes, let me rebase.

Reply:
Oh wait we need to upgrade images, do we? Probably a followup, yeah. Let me create a ticket:

https://github.com/osbuild/image-builder-cli/issues/216